source/javascript/components/accounts/imex.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+angular
+    .module('cloud-inquisitor.components')
+    .component('accountImportExport', {
+        bindings: {
+            onImport: '<',
+            onExport: '<',
+            params: '<',
+            result: '<'
+        },
+        controller: AccountImportExportController,
+        controllerAs: 'vm',
+        templateUrl: 'accounts/imex.html'
+    })
+;
+
+AccountImportExportController.$inject = ['$scope', 'Utils', '$mdDialog'];
+function AccountImportExportController($scope, Utils, $mdDialog) {
+    const vm = this;
+    vm.importData = {
+        percent: 0,
+        content: null
+    };
+    vm.onDrop = onDrop;
+    vm.onDragOver = onDragOver;
+    vm.importAccounts = importAccounts;
+    vm.$onInit = onInit;
+
+    function onInit() {
+        const drop = $('#accountData')[0];
+        drop.addEventListener('drop', vm.onDrop);
+        drop.addEventListener('dragover', vm.onDragOver);
+    }
+
+    function onLoadEnd(evt) {
+        vm.importData.percent = 100;
+        vm.importData.content = Utils.toPrettyJSON(evt.target.result);
+        $scope.$apply();
+    }
+
+    function onDragOver(evt) {
+        evt.preventDefault();
+    }
+
+    function onDrop(evt) {
+        evt.stopPropagation();
+        evt.preventDefault();
+
+        const files = evt.dataTransfer.files;
+        let reader = new window.FileReader();
+        reader.onloadend = onLoadEnd;
+
+        if (files.length !== 1) {
+            Utils.toast('You can only load a single file at a time', 'error');
+        } else {
+            const f = files[0];
```

Comment:
Please don't use single letter variables. Can we change `f` -> `file` or `_file` something that isn't a single letter and is descriptive.